build.gradle.kts

```diff
@@ -0,0 +1,201 @@
+import org.ajoberstar.grgit.Grgit
+import com.github.spotbugs.*
+
+buildscript {
+    repositories {
+        jcenter()
+        maven {
+            setUrl("https://github.com/WPIRoboticsProjects/opencv-maven/raw/mvn-repo")
+        }
+    }
+    dependencies {
+        classpath(group = "de.dynamicfiles.projects.gradle.plugins", name = "javafx-gradle-plugin", version = "8.8.2")
+        classpath(group = "edu.wpi.first.wpilib.opencv", name = "opencv-installer", version = "2.0.0")
+    }
+}
+
+plugins {
+    `java`
+    `jacoco`
+    `checkstyle`
+    `pmd`
+    id("com.github.johnrengelman.shadow") version "4.0.3"
+    id("com.google.osdetector") version "1.4.0"
+    id("org.ajoberstar.grgit") version "2.0.0" apply false
+    id("net.ltgt.errorprone") version "0.0.16"
+    id("com.github.spotbugs") version "1.6.4"
+}
+
+repositories {
+    mavenCentral()
+    jcenter()
+}
+
+tasks.withType<Wrapper>().configureEach {
+    gradleVersion = "5.0"
+    distributionType = Wrapper.DistributionType.ALL
+}
```

Comment:
I usually keep this at the bottom. Why `5.0` instead of `5.1`? The memory leak issue?